pac4j-openid/pom.xml

```diff
@@ -35,18 +40,27 @@
         <dependency>
             <groupId>org.openid4java</groupId>
             <artifactId>openid4java</artifactId>
+            <version>${openid4java.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
If I exclude the `commons-logging` dependency for `openid4java`, the `jcl-over-slf4j` is mandatory, isn't it? So I shouldn't mark it as `provided`, should I?